lib/msal-common/src/uCache/serialize/Serializer.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccessTokenEntity } from "../entities/AccessTokenEntity";
+import { IdTokenEntity } from "../entities/IdTokenEntity";
+import { RefreshTokenEntity } from "../entities/RefreshTokenEntity";
+import { AccountEntity } from "../entities/AccountEntity";
+import { AppMetadataEntity} from "../entities/AppMetadataEntity";
+import { CacheHelper } from "../utils/CacheHelper";
+import {
+    AccessTokenCacheMaps,
+    IdTokenCacheMaps,
+    RefreshTokenCacheMaps,
+    AccountCacheMaps,
+    AppMetadataCacheMaps
+} from "./JsonKeys";
+
+export class Serializer {
+    /**
+     * generate an access token CacheEntity
+     * @param accessToken
+     */
+    static serializeAccessTokenEntity(accessToken: AccessTokenEntity) {
+        const mappedAT: string = CacheHelper.renameKeys(
+            accessToken,
+            AccessTokenCacheMaps.toCacheMap
+        );
+        const atKey: string = accessToken.generateAccessTokenEntityKey();
+
+        const atCacheEntity: any = {};
+        atCacheEntity[atKey] = mappedAT;
+        return atCacheEntity;
+    }
+
+    /**
+     * generate an id token CacheEntity
+     * @param idToken
+     */
+    static serializeIdTokenCacheEntity(idToken: IdTokenEntity) {
+        const mappedIdToken: string = CacheHelper.renameKeys(
+            idToken,
+            IdTokenCacheMaps.toCacheMap
+        );
+        const idTokenKey: string = idToken.generateIdTokenEntityKey();
+
+        const idTokenCacheEntity: any = {};
+        idTokenCacheEntity[idTokenKey] = mappedIdToken;
+        return idTokenCacheEntity;
```

Comment:
Would prefer not to see `any` (hopefully will forbid this via the tsc compiler or lint).

```js
return {
  [idTokenKey]: mappedIdToken
}
```

Reply:
yep. I added types wherever I can, will revisit this.

Reply:
Cool. I didn't know we can do that. Modified now.

Reply:
Done.